mmv1/third_party/validator/tests/data/example_filestore_backup.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
remove `mmv1/third_party/validator/tests/data/example_filestore_backup.json` and `mmv1/third_party/validator/tests/data/example_filestore_backup.tf`, you don't need to have any validator related tests